src/erlfmt.erl

```diff
@@ -68,8 +70,12 @@ format_file(FileName, {Pragma, Out}) ->
     try
         case file_read_nodes(FileName, Pragma) of
             {ok, Nodes, Warnings} ->
-                [$\n | Formatted] = format_nodes(Nodes),
-                verify_nodes(FileName, Nodes, Formatted),
+                NodesWithPragma = case Pragma of
+                    insert -> insert_pragma_nodes(Nodes);
+                    _ -> Nodes
+                end,
```

Comment:
What do you think about folding this check into `file_read_nodes`? We could add another clause here: https://github.com/WhatsApp/erlfmt/pull/63/files#diff-b4326b25dd3afce6a964f37ad5778582R207 with something like `false when Pragma =:= insert`

Reply:
I think then we would have to come up with a new name for file_read_nodes that makes it clear that it will also edit the nodes.  I cannot think of a good name, which is typically a warning sign, so I will keep this outside.  But if you come up with a name, please let me know.